### DISCUSSION TITLE
articles/linux/openwrt

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#25] [General]
# articles/linux/openwrt

OpenWRT 我之前买了一个 n1 盒子用来玩，上面已经刷了 iStoreOS，这是一个 OpenWRT 的分支。这里记录了一些我的折腾过程。 日后我可能会为其刷其他系统（Arch），不过现在还是先算了吧。 包管理 OpenWRT 用的包管理器是 opkg。虽然 OpenWRT 已经声称要转到 apt，但是还没转，并且 iStoreOS 应该还没有那...

https://absx.pages.dev/articles/linux/openwrt.html

**Comment by ericzhang-debug** [comment]
来玩RouterOS和iKuai

**Reply by lxl66566** [reply]
[已经](https://t.me/withabsolutex/2184)刷了 ArchLinux ARM，回归原始
